ccd-definition/CaseEvent.json

```diff
@@ -28,7 +28,7 @@
     "SecurityClassification": "Public",
     "ShowSummary": "N",
     "ShowEventNotes": "N",
-    "EndButtonLabel": "Save and continue"    
+    "EndButtonLabel": "Save and continue"
```

Comment:
@Jamesnelson117 what do you think about having some sort of linter that makes sure no trailing spaces are left?